generated/nidaqmx/system/physical_channel.py

```diff
@@ -1218,20 +1247,22 @@ def configure_teds(self, file_path=""):
             with cfunc.arglock:
                 if cfunc.argtypes is None:
                     cfunc.argtypes = [
-                        ctypes_byte_str, ctypes_byte_str]
+                        ctypes_byte_str, ctypes_byte_str, ctypes_byte_str]
 
         error_code = cfunc(
-            self._name, file_path)
+            self._name, physical_channel, file_path)
         check_for_error(error_code)
 
     def write_to_teds_from_array(
-            self, bit_stream=None,
+            self, physical_channel="", bit_stream=None,
```

Comment:
Removing `device_name`fix is dependent on metadata functions.py from this https://github.com/ni/grpc-device-scrapigen/pull/150 and fix logic will be added once when that metadata finalize.

Reply:
Resolved.